hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/execution/traceability/HederaTracer.java

```diff
@@ -363,4 +365,14 @@ private static <E, I> String get(
             final E subject, final Function<E, I> getter, final Function<I, String> processor) {
         return null != subject ? processor.compose(getter).apply(subject) : "null";
     }
+
+    private EntityId getEntityIdOrNullByAddressAndMessageFrame(
+            Address address, MessageFrame frame, SolidityAction action) {
+        try {
+            return EntityId.fromAddress(asMirrorAddress(address, frame));
+        } catch (IllegalArgumentException e) {
+            action.setTargetedAddress(address.toArray());
```

Comment:
Steven suggested we add this line so that the user will get some idea of who the recipient was if the lookup fails.